es-vue-base/src/lib-icons/icon-contact-us.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <svg
+        :style="{
+            height: height,
+            width: width,
+        }"
+        viewBox="0 0 48 31"
```

Comment:
The `viewBox` of our icons should be relatively consistent. Most are expected to be `0 0 24 24`, with some exceptions being `0 0 32 32`.

Unfortunately the icons here are nether of those and are not uniform amongst themselves. Please ask design for corrected sizes in one of those two viewBox definitions.